src/elm/Session/Guest.elm

```diff
@@ -114,6 +136,20 @@ view thisMsg _ ({ shared } as model) content =
                 ]
 
 
+viewLeftCol : String -> Html msg
+viewLeftCol mdWidth =
+    div
+        -- Left part with background and quote (desktop only)
+        [ class "hidden md:block md:visible min-h-screen bg-bottom bg-no-repeat"
+        , class mdWidth
+        , style "background-color" "#EFF9FB"
+        , style "background-size" "auto 80%"
+        , style "background-image" "url(images/auth_bg_full.png)"
+        ]
+        [-- Use `viewQuote` with actual data here to show the user's quote
```

Comment:
we still don’t have actual quotes to put in here ):

Reply:
It looks like this comment is a bit misleading. I removed it and have put the [link to quotes-related issue](https://github.com/cambiatus/frontend/issues/238) to the `viewQuote` function description. I guess it will be more straightforward for future us.